docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "2"
+services:
+  build-test:
+    build:
+      context: ./ops/ci
+      dockerfile: Dockerfile.node-alpine_with_glibc
```

```suggestion
    image: node:12
```
That said, if it's possible to bump this above 12 - ideally as high as 16 - that would be great. 12 is already in maintenance LTS and will be out of support in 6mo: https://nodejs.org/en/about/releases/